src/test/java/org/threeten/extra/chrono/TestInternationalFixedChronology.java

```diff
@@ -0,0 +1,1184 @@
+            {2012, 3, 28, DAY_OF_YEAR, 366, 2012, 0, 0},
+
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_MONTH, 0, 2012, -1, -1},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_MONTH, 1, 2012, 7, 1},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_MONTH, 2, 2012, 7, 2},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_MONTH, 3, 2012, 7, 3},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_MONTH, 4, 2012, 7, 4},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_MONTH, 5, 2012, 7, 5},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_MONTH, 6, 2012, 7, 6},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_MONTH, 7, 2012, 7, 7},
+
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_YEAR, 0, 2012, -1, -1},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_YEAR, 1, 2012, 7, 1},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_YEAR, 2, 2012, 7, 2},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_YEAR, 3, 2012, 7, 3},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_YEAR, 4, 2012, 7, 4},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_YEAR, 5, 2012, 7, 5},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_YEAR, 6, 2012, 7, 6},
+            {2012, -1, -1, ALIGNED_DAY_OF_WEEK_IN_YEAR, 7, 2012, 7, 7},
+
+            {2012, -1, -1, ALIGNED_WEEK_OF_MONTH, 0, 2012, -1, -1},
+            {2012, -1, -1, ALIGNED_WEEK_OF_MONTH, 3, 2012, 7, 15},
+
+            {2012, -1, -1, ALIGNED_WEEK_OF_YEAR, 0, 2012, -1, -1},
+            {2012, -1, -1, ALIGNED_WEEK_OF_YEAR, 3, 2012, 1, 15},
+
+            {2012, -1, -1, DAY_OF_WEEK, 0, 2012, -1, -1},
+            {2012, -1, -1, DAY_OF_WEEK, 1, 2012, 7, 1},
+            {2012, -1, -1, DAY_OF_WEEK, 2, 2012, 7, 2},
+            {2012, -1, -1, DAY_OF_WEEK, 3, 2012, 7, 3},
+            {2012, -1, -1, DAY_OF_WEEK, 4, 2012, 7, 4},
+            {2012, -1, -1, DAY_OF_WEEK, 5, 2012, 7, 5},
+            {2012, -1, -1, DAY_OF_WEEK, 6, 2012, 7, 6},
+            {2012, -1, -1, DAY_OF_WEEK, 7, 2012, 7, 7},
+
+            {2012, -1, -1, DAY_OF_MONTH, 0, 2012, 0, 0},
+            {2012, -1, -1, DAY_OF_MONTH, -1, 2012, -1, -1},
+            {2012, -1, -1, DAY_OF_MONTH, 3, 2012, 7, 3},
+
+            {2012, -1, -1, MONTH_OF_YEAR, -1, 2012, -1, -1},
+            {2012, -1, -1, MONTH_OF_YEAR, 0, 2012, 0, 0},
+            {2012, -1, -1, MONTH_OF_YEAR, 7, 2012, 7, 1},
+            {2012, -1, -1, MONTH_OF_YEAR, 2, 2012, 2, 1},
+
+            {2012, -1, -1, YEAR, 2012, 2012, -1, -1},
+            {2012, -1, -1, YEAR, 2013, 2013, 7, 1},
+            {2012, -1, -1, YEAR, 2011, 2011, 7, 1},
+            {2012, -1, -1, YEAR, 2016, 2016, -1, -1},
+
+            {2012, 3, 28, DAY_OF_MONTH, -1, 2012, -1, -1},
+            {2012, 1, 28, DAY_OF_MONTH, -1, 2012, -1, -1},
+            {2012, 3, 28, MONTH_OF_YEAR, -1, 2012, -1, -1},
+            {2012, 3, 28, DAY_OF_YEAR, 169, 2012, -1, -1},
+            {2013, 3, 28, DAY_OF_YEAR, 169, 2013, 7, 1},
+            {2013, 7, 1, YEAR, 2012, 2012, 7, 1},
+        };
```

Comment:
I recommend you add these tests at the bottom of the `with(...)` data 

     {2012, 1, 1, ALIGNED_WEEK_OF_YEAR, 52, 2012, 13, 22},
     {2012, 13, 28, ALIGNED_WEEK_OF_YEAR, 1, 2012, 1, 7},

Then work the error that appears.